parcel_test.go

```diff
@@ -28,60 +29,93 @@ func getTestParcel() Parcel {
 	}
 }
 
+// setupStore создает подключение к БД и возвращает store
+func setupStore(t *testing.T) ParcelStore {
+	db, err := sql.Open("sqlite", "tracker.db")
+	require.NoError(t, err, "No db connection")
+
+	t.Cleanup(func() {
+		db.Close()
+	})
+
+	return NewParcelStore(db)
+}
+
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
-	store := NewParcelStore(db)
+	store := setupStore(t)
 	parcel := getTestParcel()
 
 	// add
-	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	number, err := store.Add(parcel)
+	require.NoError(t, err, fmt.Sprintf("Errow adding parcel: %s\n", err))
+	require.NotEmpty(t, number, "id is empty")
 
 	// get
-	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	p, err := store.Get(number)
+	require.NoError(t, err, fmt.Sprintf("Parcel with id: %d not found, error: %s\n", number, err))
+
+	require.Equal(t, parcel.Client, p.Client, fmt.Sprintf("parcel client mismatch, expected: %d, returned: %d", parcel.Client, p.Client))
```

Comment:
`require.Equal` Умеет сравнивать структуры и не надо каждое поле проверять
```suggestion
	require.Equal(t, parcel, p)
```

Reply:
но в таком случае тест всегда будет FAIL, потому что он будет пытаться сравнить автоинкрементируемое поле parcel.ID

Reply:
Можно в 51 строчке убрать переменную `number` и присвоить `parcel.Number, err := store.Add(parcel)`, тогда айди будет нужный.

А при таком подходе если в структуре поменяется/добавится поле, то нужно будет все тесты менять